src/key_io.cpp

```diff
@@ -142,7 +142,9 @@ CKey DecodeSecret(const std::string& str)
             key.Set(data.begin() + privkey_prefix.size(), data.begin() + privkey_prefix.size() + 32, compressed);
         }
     }
-    memory_cleanse(data.data(), data.size());
+    if (data.data()) {
+        memory_cleanse(data.data(), data.size());
+    }
```

Comment:
This check is no longer needed?

Reply:
Notes
If size() is 0, data() may or may not return a null pointer.
https://en.cppreference.com/w/cpp/container/vector/data